### PR TITLE
arm64: boot: dts: qcom: sdm632-motorola-ocean: Add backlight

### PR DESCRIPTION
**Pull request by Ultracoolguy**

### CODE REVIEW

File: arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts [hunk 1]
```diff
@@ -132,6 +132,24 @@
...
+	led-controller@36 {
+		compatible = "ti,lm3697";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		reg = <0x36>;
+
+		led@1 {
+			reg = <1>;
+			led-sources = <0 1 2>;
+			label = "white:backlight";
```
Comment (z3ntu) [inline arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts:147]: Please try replacing the label with
```
function = LED_FUNCTION_BACKLIGHT;
color = <LED_COLOR_ID_WHITE>;
```

Comment (Ultracoolguy) [reply arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts:147]: If I do that dtc gives me syntax error. Also according to Documentation/leds-class.rst and Documentation/devicetree/bindings/leds/leds-lm3697.txt(or at least from what I can tell), the label method is the proper way of doing it.

Comment (z3ntu) [reply arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts:147]: `label` is deprecated, see https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/leds/common.yaml#L54-L60

You need to include `#include <dt-bindings/leds/common.h>` at the top for the function & color constants

Comment (Ultracoolguy) [reply arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts:147]: Ahh, now it compiled. Now I just have to test in my phone is it works and then push.

Thank you!

Comment (Ultracoolguy) [reply arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts]: It worked, so ig I'm resolving this conversation.

Once again, ty!

Comment (z3ntu) [reply arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts]: Great :)

**Pull request merged**